owncloudTestUtil/src/main/java/com/owncloud/android/testutil/OCUser.kt

```diff
@@ -29,6 +29,7 @@ val OC_USER_INFO = UserInfo(
 )
 
 val OC_USER_QUOTA = UserQuota(
+    accountName = "",
```

Comment:
Use `OC_ACCOUNT_NAME` instead. That's the dummy one we use for the tests. Same for other tests in the PR 👍